examples/README.md

```diff
@@ -12,7 +12,8 @@ application runs on a trusted server and can protect secret data, such
 as the CLIENT_SECRET variable.
 
 To use:
-1. `yarn install`
+1. `cd fhir-kit-client && yarn install`
+1. `cd examples && yarn install`
```

Comment:
How about:

In the repo root, run `yarn install`
In the example directory, run `yarn install`

Reply:
It's actually run the install command in each example subdirectory. There is no `package.json` in `/examples`.